arbos/chains.go

```diff
@@ -19,11 +28,31 @@ func getStaticChainConfig(chainId *big.Int) (*params.ChainConfig, error) {
 	return nil, fmt.Errorf("unsupported L2 chain ID %v", chainId)
 }
 
-func GetChainConfig(chainId *big.Int, genesisBlockNum uint64) (*params.ChainConfig, error) {
-	staticChainConfig, err := getStaticChainConfig(chainId)
-	if err != nil {
-		return nil, err
+func GetChainConfig(chainId *big.Int, genesisBlockNum uint64, l2ChainInfoFiles []string) (*params.ChainConfig, error) {
+	for _, l2ChainInfoFile := range l2ChainInfoFiles {
+		chainsInfoBytes, err := os.ReadFile(l2ChainInfoFile)
+		if err != nil {
+			return nil, err
```

Comment:
We should put the path in the error message here. A generic "file not found" would be confusing to users.